gapis/api/vulkan/api/query_pool.api

```diff
@@ -110,12 +110,13 @@ cmd void vkDestroyQueryPool(
 @threadSafety("system")
 @indirect("VkDevice")
 @threadsafe
-@no_replay
+//@no_replay
 // GetQueryPoolResults has no semantic impact
 // on replay, so avoid replaying it. It can cause
 // hangs depending on the state of the query pool
 // TODO(awoloszyn): Work out all of the cases where this
```

Comment:
You can remove this comment as we believe we are doing this correctly.

Reply:
done